.github/actions/templates/avm-setEnvironment/action.yml

```diff
@@ -45,7 +45,7 @@ runs:
 
         # Define PS modules to install on the runner
         $modules = @(
-            @{ Name = 'Az.Accounts' },
+            @{ Name = 'Az.Accounts'; Version = '4.0.2' },
```

Comment:
Please add a comment here to explain why this version so if we ever want to allow any never version than 4.0.2 and retest we know which module this was for. The same applies to the other explicit version your hardcoded below.

Reply:
Resolved

Reply:
This has been removed as its not needed anymore, it appears the agents are now up-to-date with the latest